assets/xml/objects/object_boss01.xml

```diff
@@ -113,7 +113,7 @@
         <Limb Name="object_boss01_Standardlimb_00EFB8" Type="Standard" EnumName="OBJECT_BOSS01_1_LIMB_31" Offset="0xEFB8" />
         <Limb Name="object_boss01_Standardlimb_00EFC4" Type="Standard" EnumName="OBJECT_BOSS01_1_LIMB_32" Offset="0xEFC4" />
         <Limb Name="object_boss01_Standardlimb_00EFD0" Type="Standard" EnumName="OBJECT_BOSS01_1_LIMB_33" Offset="0xEFD0" />
-        <Skeleton Name="object_boss01_Skel_00F0A8" Type="Flex" LimbType="Standard" LimbNone="OBJECT_BOSS01_1_LIMB_NONE" LimbMax="OBJECT_BOSS01_1_LIMB_MAX" EnumName="object_boss01_1_Limbs" Offset="0xF0A8" />
+        <Skeleton Name="object_boss01_Skel_00F0A8" Type="Flex" LimbType="Standard" LimbNone="OBJECT_BOSS01_1_LIMB_NONE" LimbMax="OBJECT_BOSS01_1_LIMB_MAX" EnumName="ObjectBoss011Limbs" Offset="0xF0A8" />
```

Comment:
Do we care for an exception for the pascal case here or is the `Boss011` okay?

Reply:
Actually coming back to this it's not a huge deal, it'll be "fixed" once the actor is documented more

Reply:
Yeah, I'm not a huge fan of this, but at the end of the day this is a temporary name. Once the object is actually documented, this will go away.

Reply:
For what it is worth the same thought went through my head as I was fixing these. However it is temporary, and I can't really think of a good solution other then just wait until they renamed.

Reply:
Speaking of which, shouldn't this be `ObjectBoss011Limb` instead? (without the "s")

Reply:
Yes, yes it should. Fixed.